static/src/javascripts/boot.js

```diff
@@ -38,7 +39,12 @@ const go = () => {
         const pageViewId: ?string = config.get('ophan.pageViewId');
         const pubData: { browserId?: ?string, pageViewId?: ?string } =
             { browserId, pageViewId };
-        cmp.init({ pubData, isInUsa: isInUsa() });
+
+        if (config.get('tests.useAusCmpVariant') === 'variant') {
```

Comment:
@mxdvl simplified this logic per our chat

Reply:
For people outside the test, the old interface is used, still.